docs/developer-portal.mdx

```diff
@@ -123,10 +123,11 @@ ___
 - <ExternalLink text="Michelson Emacs Mode" link="https://github.com/tezos/tezos/tree/master/emacs" />: Michelson Plugin for Emacs
 - <ExternalLink text="Granary" link="https://stove-labs.github.io/granary/" />: Tezos Development Toolkit
 - <ExternalLink text="Truffle" link="https://www.trufflesuite.com/docs/tezos/truffle/quickstart" />: Development environment, testing framework, and asset pipeline.
-- <ExternalLink text="Tezter" link="https://github.com/Tezsure/Tezster-CLI" />: Testing tool for Tezos
+- <ExternalLink text="Tezster" link="https://github.com/Tezsure/Tezster-CLI" />: Testing tool for Tezos
 - <ExternalLink text="Magic's Testnet Faucet" link="https://magic-tezos-faucet.now.sh/" />: A simple faucet for a Tezos Testnet
 - <ExternalLink text="Testnet Faucet" link="https://faucet.tzalpha.net/" />: Faucet for a Tezos testnet
 - <ExternalLink text="Mainnet Faucet" link="https://faucet.tezos.com/" />: Faucet for the Tezos Mainnet
+- <ExternalLink text="Testnet Faucet Bot" link="tg://resolve?domain=tezos_faucet_bot" />: Telegram Faucet Bot for Tezos testnets
```

Comment:
This link doesn't work when opened on a desktop computer. Perhaps https://t.me/tezos_faucet_bot would be more appropriate.

Reply:
I agree, thanks, updated now. had copied it from the bakingbad website.